.pre-commit-config.yaml

```diff
@@ -0,0 +1,37 @@
+# See https://pre-commit.com for more information
+# See https://pre-commit.com/hooks.html for more hooks
+exclude: ^tests/integration|\.patch$
```

Comment:
jefferson doesn't have any tests, this should not be included as it has no effect.